tools/Makefile

```diff
@@ -1,28 +1,55 @@
 all: failpoint-ctl gocovmerge goveralls golangci-lint mockgen protoc-gen-gogofaster protoc-gen-grpc-gateway statik gofumports
 
-failpoint-ctl: 
-	go build -o bin/$@ github.com/pingcap/failpoint/failpoint-ctl
+failpoint-ctl:
+	@if [ ! -e ./bin/failpoint-ctl ]; then \
```

Comment:
what if the source code of failpoint-ctl is updated 🤔

Reply:
you are right, already undo this change